docs/use-mainnet/info-contracts.md

```diff
@@ -84,11 +84,11 @@ If your dapp is using the public endpoint `https://rpc.goerli.linea.build`, it m
   </tr>
   <tr>
     <td>Linea rollup and L1Bridge</td>
-    <td><a href="https://etherscan.io/address/0x4c8d4Ce72afAA417d1F7E833725FdB4E793cd6b3">0x4c8d4Ce72afAA417d1F7E833725FdB4E793cd6b3</a></td>
+    <td><a href="https://etherscan.io/address/0xd19d4B5d358258f05D7B411E21A1460D11B0876F">0xd19d4B5d358258f05D7B411E21A1460D11B0876F</a></td>
   </tr>
   <tr>
     <td>L2Bridge</td>
```

Comment:
Can we also rename it L2 Message Service to be consistent?

Reply:
done